codegen-server-test/build.gradle.kts

```diff
@@ -24,6 +24,10 @@ data class CodegenTest(val service: String, val module: String, val extraConfig:
 
 val CodegenTests = listOf(
     CodegenTest("com.amazonaws.simple#SimpleService", "simple"),
+    // TODO: this is kept here commented to allow to add the RestJson protocol test
+    // quickly when continuting the test development. As of now the tests do not generate
+    // valid tests and will fail the build.
+    // CodegenTest("aws.protocoltests.restjson#RestJson", "rest_json"),
```

Comment:
I would probably try to at least have them `check'd` in CI. You could generate tests that all have `#[ignore]`, eg.

Reply:
Done!